src/form/DecompositionTable/DecompositionTableStyles.ts

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components';
+import { Table, ContextMenu, themeColor } from '@amsterdam/asc-ui';
+
+export const StyledDecompositionTable = styled(Table)`
+	width: 100%;
+	border-collapse: collapse;
+	color: ${themeColor('tint', 'level6')};
+
+	thead {
+		background-color: ${themeColor('tint', 'level7')};
+		color: ${themeColor('tint', 'level1')};
+	}
+
+	.row {
+		background-color: rgba(0, 70, 153, 0.2);
+
+		&__unit {
+			background-color: ${themeColor('tint', 'level3')};
+		}
+
+		&__manifestation {
+			background-color: ${themeColor('tint', 'level1')};
+		}
+	}
+
+	.centered {
+		text-align: center;
+	}
+
+	input[role='switch']:checked + span {
+		background-color: ${themeColor('primary')};
+		border-color: ${themeColor('primary')};
+	}
+
+	button {
+		border: none;
+		background-color: transparent;
+		color: ${themeColor('primary')};
+	}
+
+	button svg {
+		fill: ${themeColor('primary')};
+	}
+`;
+
+export const StyledContextMenu = styled(ContextMenu)`
```

Comment:
I'd suggest putting this in a place dedicated to the context menu component. If you need an override on those styles specific to the decomposition table component you can override styles with a higher specificity: https://styled-components.com/docs/faqs#how-can-i-override-styles-with-higher-specificity